### ISSUE TITLE
Template doesn't render graph if a different hostname is used

### ISSUE THREAD

**Issue by bergos** [zazukoians/trifid-ld#13]
There is no data on /resource/Eiffel_Tower on the html page, but the graph data is embedded in the html. In the configuration the dbpedia SPARQL endpoind is used, but the request goes to http://localhost:8080/resource/Eiffel_Tower. Here the example [config.js](https://gist.github.com/bergos/b8a053699feeb7ff6d84).

**Comment by retog** [comment]
If trifid dereferences http://somehost/resource/Eifel_Tower to the description of the resourse http://someotherhost/resource/Eifel_Tower it should either replace the prefixes (i.e. replace "http://someotherhost/" with "http://somehost/resource/) or add a statement pointing from http://somehost/resource/Eifel_Tower to http://someotherhost/resource/Eifel_Tower such as an owl:sameAs statement.

**Comment by bergos** [comment]
I expect the problem also occurs if there are triples that are not linked to the main subject. It could be confusing if triples are missing. Sure, this would be a data quality problem, but the interface should help to identify such problems.

The old React component looped over all unique subjects and created a table for each of it. The RDF2H template should do it the same way.

**Comment by retog** [comment]
With RDF2h only connected graphs can be rendered. Since a resource description is always a connected graph we have no issue here. If there are detached triples it's not a problem with the data quality but something went wrong somewhere else in trifid.

**Comment by ktk** [comment]
I'm not really sure if I get that. In the case of dereferencing I would expect to get a resource back which is the URI I was talking about. But in the case of an LDP container I could get anything back which quite probably has nothing to do with the URI of the container itself.

So what is the statement here, RDF2h cannot do that or in this particular case it doesn't do that? When I think back I need to specify a URI as entry point, which is typically either a resource or a class right? So in this case the problem would be that I neither know the resource URI nor the class right?

**Comment by retog** [comment]
@ktk you're right with LDP or SPARQL graphstore RDF2h cannot render what the server returns as this might be a disconnected graph and might even not mention the derefernced resource. But I think this has nothing to do with trifid which is about resources.
